vangoh.go

```diff
@@ -341,7 +355,17 @@ func (vg *Vangoh) CreateSigningString(r *http.Request) string {
 	buffer.WriteString(r.Header.Get("Content-Type"))
 	buffer.WriteString(newline)
 
-	buffer.WriteString(r.Header.Get("Date"))
+	dateHeader := ""
```

Comment:
Since this logic is duplicated, maybe write a helper function for getting the date header from a `request`? Easier to test, too.